src/app/clusters/audio-output-server/audio-output-server.cpp

```diff
@@ -41,16 +41,21 @@
 #include <app/Command.h>
 #include <app/util/af.h>
 
-bool emberAfAudioOutputClusterRenameOutputCallback(chip::app::Command * commandObj, unsigned char, unsigned char *)
+bool audioOutputClusterSelectOutput(uint8_t index);
+bool audioOutputClusterRenameOutput(uint8_t index, uint8_t * name);
```

Comment:
Callbacks zapt as followup.

Reply:
Issue [link](https://github.com/project-chip/connectedhomeip/issues/8086) for follow up.